step.yml

```diff
@@ -163,7 +163,7 @@ inputs:
       is_required: true
   - test_repetition_mode: "none"
     opts:
-      title: Test Repetition Mode
+      title: Test Repetition Mode (Available in Xcode 13+)
```

Comment:
I understand you wanted to highlight the availability of this feature in the title but it would be better to put a heads-up on it in the description: "Please note that the Test Repetition Mode is only available in Xcode 13+."

Reply:
I think there are pros and cons of both approaches. In this case, I'd suggest to keep it in the title for the following reasons:
* Xcode 13 has not even been released yet. When it becomes available (this fall), it will take some time for developers to migrate to. Knowing that something is available only on the newest stack/Xcode version is vital information that I think is worth mentioning in the title.
* Keeping up with the availability of features (both framework and tooling) is already a bit of pain for iOS developers; I think having an overview of what's available to you and what is not just by looking at the titles of the step inputs can be very helpful.
* Since we expose step inputs _regardless_ of the stack they are running on (so all inputs are visible/usable on all stacks), having this overview of availability (as I mentioned above) can be very helpful. You don't have to peak into every input's description just to get the information of where it is available.

What do you think?

Reply:
I like your reasoning, we can follow what works best for our users! ;)

Reply:
Cool!

Reply:
I would vote for the current highlight (in the title) because the description is only visible in the workflow editor if the user expands the summary by clicking on it. It's not the best UX and I think a lot of people would miss this important detail if it was visible only in the input description.